### DISCUSSION TITLE
Include ERC-20 asset address in cancel and withdraw events

### DISCUSSION THREAD

**Discussion by PaulRBerg** [sablier-labs/evm-monorepo#701] [Ideas]
The lack of the ERC-20 asset in these events makes it impossible to build certain data queries on Dune, e.g. "what is the volume of stablecoin withdrawals".

https://github.com/sablier-labs/v2-core/blob/8ca98d3ffad2dd75e0a705ffc0cd5e798acc8058/src/interfaces/ISablierV2Lockup.sol#L29-L35

https://github.com/sablier-labs/v2-core/blob/8ca98d3ffad2dd75e0a705ffc0cd5e798acc8058/src/interfaces/ISablierV2Lockup.sol#L53

**Comment by andreivladbrg** [comment]
Let's do it, there would only be a slight increase in the gas cost, which is totally fine.

**Reply by PaulRBerg** [reply]
Cool
